setup.py

```diff
@@ -14,7 +14,7 @@ def read(fname):
 
 
 def desc():
-    info = read('README.rst')
+    info = read("README.md")
```

Comment:
[Setting long_description_content_type is required for Markdown](https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata) as the [spec is to default to RST and fallback to plain text](https://packaging.python.org/en/latest/specifications/core-metadata/#description-content-type)